serverless.yml

```diff
@@ -0,0 +1,38 @@
+service: flask-api-lambda
+
+provider:
```

Comment:
To be honest, I'm not sure what any of this stuff means, but looking into AWS might help me with that.

Reply:
I am in the same situation Ramiz:) But good work Myles everything looks good, I don't see any errors or issues so far!